CAJA_WS.php

```
@@ -1197,7 +1197,7 @@ function isExtensionAllowed($filename, $mediaOnly = false) {
 	if ($mediaOnly) {
 		$allowed = $media;
 	} else {
-		$allowed = array_merge($media, $other);
+		$allowed = $media ? array_merge($media, $other) : $other;
```

this is to compensate for the possible empty array, correct? Really this condition should throw an error

---

fixed with your PR